.github/release.yml

```diff
@@ -0,0 +1,15 @@
+changelog:
+  categories:
+    - title: Testable features, fixes, enhancements 🎉
+      labels:
+        - '*'
+      exclude:
+        labels:
+        - not-testable
+        - Nuernberg
+    - title: Testable features for nuernberg 🎉
+      labels:
+        - Nuernberg
```

Comment:
:thinking: Why do we only need this for Nürnberg but not for Bayern/Koblenz? And I think that we have to be careful when assigning labels to PRs this way that we base it on which projects the changes affect instead of who hired/pays us to implement the feature.

Reply:
Because we only have to inform nuernberg about some  important changes. Currently no other customer requested that

Reply:
I still think it would be helpful for both Martin, people from other teams and even us to see at a glance which changes are only related to a specific project. Also it feels inconsistent to only do this for Nuernberg.

Reply:
I don't think this will work, since changelog items won't be duplicated and some changes may not only be related to nuernberg but nuernberg may want to test them.
If one changelog item applies to a category it won't be shown in another again. 
We just highlight here changes that nuernberg may want to test but may be also related to other projects

Reply:
For me thats even more reason to specify the other projects as well, such that we can tell Nuernberg that everything for all projects and Nuernberg is relevant? Or did you plan to label every single PR with Nuernberg, even if it affects the other projects as well? I am a little confused here, probably I am just not getting it though.

Reply:
Ok it looks like you didn't understand the mechanism how the release notes work, so i will try to explain it.
Lets say we have feature that affects nuernberg and koblenz.
Then we would add both project labels in the pr.
The release note template will take the first category listed in the template that fits to one label

So lets say we create a category for nuernberg that only shows prs with nuernberg label und do the same for koblenz. Then the pr that is marked with both labels will only appear for nuernberg.

So yes we tag every single pr with nuernberg that affects nuernberg where we think nuernberg should be informed about this change because sth. important might break

Reply:
> So yes we tag every single pr with nuernberg that affects nuernberg where we think nuernberg should be informed about this change because sth. important might break

So, we are making Nürnberg oriented release notes here.

The downside of this approach would be that if we later receive the same request from Koblenz, we would have to rework the approach again.